.gitignore

```diff
@@ -23,11 +24,12 @@ coverage
 # Compiled binary addons (http://nodejs.org/api/addons.html)
 build/Release
 
-# Dependency directory
+# Dependency directories
 node_modules
+jspm_packages
```

Comment:
Spoke in person, we discussed that this was part of the github/gitignore node's version.